projects/angular/src/modal/modal-stack.service.ts

```diff
@@ -45,6 +45,10 @@ export class ModalStackService {
 
   private onKeyUp(event: KeyboardEvent) {
     if (this.modalStack.length && normalizeKey(event.key) === Keys.Escape) {
+      // We blur the active element because escaping with an input element in focus could cause
```

Comment:
Thanks for quickly resolving this issue. Can you write a unit test to ensure we don't reintroduce this issue in the future?

Reply:
Yeah, I'll try.

Reply:
I am having trouble producing the error in a unit test.

Reply:
Ok. Thanks for looking into it.